docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -1,226 +0,0 @@
-[[modules-discovery-zen]]
```

Comment:
Ditton adding to https://www.elastic.co/guide/en/elasticsearch/reference/master/redirects.html

Reply:
TIL, thanks. Done in 400b2e46d68.